x-pack/test/detection_engine_api_integration/security_and_spaces/prebuilt_rules/fleet_integration.ts

```diff
@@ -48,6 +48,8 @@ export default ({ getService }: FtrProviderContext): void => {
         overrideExistingPackage: true,
       });
 
+      await es.indices.refresh({ index: ALL_SAVED_OBJECT_INDICES });
```

Comment:
`installPrebuiltRulesFleetPackage()` [already contains](https://github.com/elastic/kibana/blob/main/x-pack/test/detection_engine_api_integration/utils/prebuilt_rules/install_prebuilt_rules_fleet_package.ts#L68) index refresh. Are you sure the second sequential index refresh is needed?